yarn.lock

```diff
@@ -3902,9 +3902,9 @@
     semver "^7.3.5"
     tsutils "^3.21.0"
 
-"@typescript-eslint/utils@5.21.0", "@typescript-eslint/utils@^5.10.0":
+"@typescript-eslint/utils@5.21.0", "@typescript-eslint/utils@^5.10.0", "@typescript-eslint/utils@^5.21.0":
   version "5.21.0"
-  resolved "https://registry.yarnpkg.com/@typescript-eslint/utils/-/utils-5.21.0.tgz#51d7886a6f0575e23706e5548c7e87bce42d7c18"
+  resolved "https://registry.npmmirror.com/@typescript-eslint/utils/-/utils-5.21.0.tgz#51d7886a6f0575e23706e5548c7e87bce42d7c18"
```

Comment:
is this npmmirror change expected?

Reply:
Oh, thanks for noticing🤦‍♂️ I usually remember to revert the resolution before pushing, but forgot this time. It will be corrected next weekend when I bump dependencies.

It's mostly benign. `npmmirror` is trustable.

If we migrate to Yarn v3 (#6028), the resolved registry URL will not be in the lockfile.